tasks/main.yml

```
@@ -1,8 +1,25 @@
 ---
-- name: Install systemd-timesyncd
+- name: Load var file with package names based on the OS type
+  ansible.builtin.include_vars: "{{ lookup('first_found', params) }}"
```

```suggestion
  ansible.builtin.include_vars: "{{ lookup('ansible.builtin.first_found', params) }}"
```

I know this is a more widespread change but I figured I would mention it here.

---

Good catch!  I copied and pasted that from elsewhere.